src/main/java/io/nats/client/impl/Headers.java

```diff
@@ -269,6 +269,17 @@ public boolean containsKeyIgnoreCase(String key) {
 		return values == null ? null : Collections.unmodifiableList(values);
 	}
 
+	/**
+	 * Returns the first value for the specific (case sensitive) key.
+	 * Will be {@code null} if the key is not found.
+	 *
+	 * @return the first value for the case sensitive key.
+	 */
+	public String getFirst(String key) {
```

Comment:
not related to KV but more in line with header ADR. Headers were released before I knew that header get was supposed to only return the top value, not the entire list. Added this as a convenience. Probably will change next time there is a SemVer change